src/app/fdctl/configure/workspace.c

```diff
@@ -281,12 +317,20 @@ init( config_t * const config ) {
         }
         break;
       case wksp_bank_shred:
+        ulong1( pod, "cnt", config->layout.bank_tile_count );
         for( ulong i=0; i<config->layout.bank_tile_count; i++ ) {
+          /* FIXME switch bank->shred for receive_buffer_size */
```

Comment:
? Don't follow what this FIXME wants

Reply:
All these mcaches are using config->tiles.bank.receive_buffer_size